ansible/vars/azure_env.yml

```diff
@@ -0,0 +1,75 @@
+---
+cloud_type: "azure"
+
+expected:
+  "1.1.1": "30000"
+  "1.1.1_runtime": "30000"
+  "1.1.2": "36000"
```

Comment:
Let's remove all the not used entries here. Put only the ones that apply to the currently added tests